core/lib/spree/testing_support/kernel.rb

```diff
@@ -0,0 +1,18 @@
+module Spree
+  module TestingSupport
+    module Kernel
+
+      private
+
+      def silence_stream(stream)
+        old_stream = stream.dup
```

Comment:
@priyank-gupta are you sure it is a good idea to copy this method into spree source? it was removed from Rails because it is not threadsafe.